azurelinuxagent/common/cgroup.py

```diff
@@ -139,19 +125,17 @@ def get_tracked_metrics(self):
 
 
 class CpuCgroup(CGroup):
-    def __init__(self, name, cgroup_path):
-        super(CpuCgroup, self).__init__(name, cgroup_path, CGroupContollers.CPU)
+    def __init__(self, name, cgroup_path, track_throttled_time=False):
```

Comment:
Out of curiosity, when will `track_throttled_time` be False? Like what's the use case of that?

Reply:
i will hook this up to the enable/disable feature... if quota is disabled then there is no need to send this metric

Reply:
(if quota is disabled we don't need this metric, but we would still be monitoring cpu usage)

Reply:
Yeah makes sense, without quota the agent wouldn't be throttled at all right? Or can it still be? Any idea?

Reply:
the quota/throttling are handled by the same controller: cpu. the cpu usage is done by a different controller: cpuacct
there can be cpu usage monitoring without quota, but throttling would be done only if a quota is set

Reply:
I got that. My main question was if there is any possibility of the agent being throttled even if there is no quota? Its more a systemd question than anything to do with this PR

Reply:
i see... the scheduler tries to allocate cpu for all processes, but it also uses other factors to  decide who gets the cpu (e.g. the nice level) so I guess that could also be seen as a form of "throttling"